CSharp/README.md

```diff
@@ -314,6 +315,84 @@ public async Task<int> GetLatestPositionAsync()
 }
 ```
 
+## Testing Code
+
+We all know testing code is not built the same way that regular code is, because its purpose is different. This is why we have special guidelines for this type of code.
+
+### Test Class naming: {TargetTestClass}Tests
+
+```csharp
+// Bad
+namespace MyProject.UnitTests
+{
+    [TestClass]
+    public class MyService
+    { }
+}
+
+// Good
+namespace MyProject.UnitTests

```

Comment:
Microsoft ASP.NET Core Engineering guidelines propose to use the same namespace of the class under test, see https://github.com/aspnet/Home/wiki/Engineering-guidelines#unit-test-class-naming

> Test class names end with `Test` and live in the same namespace as the class being tested. For example, the unit tests for the `Microsoft.Fruit.Banana` class would be in a `Microsoft.Fruit.BananaTest` class in the test assembly.

I think that it could be very useful to require less _usings_ and it helps with intellisense.